docs/source/getting_started/index.rst

```diff
@@ -0,0 +1,7 @@
+###############
+Getting Started
+###############
+
+.. mdinclude:: ../../../README.md
```

Comment:
Minor rendering error here as well - note that the last two paragraphs are indented as if they're part of the bulleted list:

![image](https://user-images.githubusercontent.com/5603551/122939935-09c2dd80-d342-11eb-8e46-0d7a638e80d8.png)